src/Facets/GnosisBridgeFacet.sol

```diff
@@ -46,16 +51,24 @@ contract GnosisBridgeFacet is ILiFi, ReentrancyGuard, SwapperV2, Validatable {
         doesNotContainDestinationCalls(_bridgeData)
         validateBridgeData(_bridgeData)
         onlyAllowDestinationChain(_bridgeData, GNOSIS_CHAIN_ID)
-        onlyAllowSourceToken(_bridgeData, DAI)
     {
-        LibAsset.depositAsset(DAI, _bridgeData.minAmount);
+        if (
+            _bridgeData.sendingAssetId != DAI &&
+            _bridgeData.sendingAssetId != USDS
+        ) {
+            revert InvalidSendingToken();
+        }
+        LibAsset.depositAsset(
+            _bridgeData.sendingAssetId,
+            _bridgeData.minAmount
+        );
         _startBridge(_bridgeData);
     }
 
-    /// @notice Performs a swap before bridging via XDaiBridge
+    /// @notice Performs a swap before bridging via BridgeRouter
```

Comment:
nitpick: Instead of BridgeRouter (which sounds super generic) I would suggest to say "via GnosisBridge" (and we use their BridgeRouter, yes...but many bridges call their contracts bridge router